.travis.yml

```diff
@@ -102,6 +113,15 @@ install:
   - if [[ $INSTALL_TYPE == 'pip_setup' ]]; then python setup.py install; fi
   - if [[ $MPI_INSTALL == '1' ]]; then pip install -e .[extras]; fi
 
+  - if [[ $DEVITO_BACKEND == 'ops' ]]; then
+      # Install CUDA
```

Comment:
would do the `export PATH` here rather than in `env`

Reply:
but there should be no OPS_INSTALL_PATH in cases where DEVITO_BACKEND != 'ops'